kayenta-stackdriver/src/test/java/com/netflix/kayenta/stackdriver/metrics/StackdriverMetricsServiceTest.java

```diff
@@ -0,0 +1,69 @@
+package com.netflix.kayenta.stackdriver.metrics;
+
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.when;
+
+import com.google.api.services.monitoring.v3.model.ListTimeSeriesResponse;
+import com.google.api.services.monitoring.v3.model.Point;
+import com.google.api.services.monitoring.v3.model.TimeSeries;
+import com.google.api.services.monitoring.v3.model.TypedValue;
+import com.netflix.kayenta.canary.CanaryConfig;
+import com.netflix.kayenta.canary.CanaryMetricConfig;
+import com.netflix.kayenta.metrics.MetricSet;
+import com.netflix.kayenta.stackdriver.canary.StackdriverCanaryScope;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
and then I believe this becomes:
```suggestion
@ExtendWith(MockitoExtension.class)
```
though I'm not sure you'll actually need it.

Reply:
Did that, but now I'm getting:

```
> Task :kayenta-stackdriver:test FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':kayenta-stackdriver:test'.
> No tests found for given includes: [StackdriverMetricsServiceTest](--tests filter)
```

Looked up the error on Google, some suggest to edit to import that what you suggested above, which I did, and some suggest an edit to the gradle.build file, which I'm not sure is necessary, and how to apply it if it is

Reply:
Aaah yes, this is a bit of a rabbit hole.  One way forward is to add
```
test {
  useJUnitPlatform()
}
```
and
```
dependencies {
  testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine"
}
```
to [kayenta-stackdriver.gradle](https://github.com/spinnaker/kayenta/blob/00b404a56687092c99dacd2eb3be1d316d396976/kayenta-stackdriver/kayenta-stackdriver.gradle).  I think that'll be enough, though I'm slightly surprised you don't also need
```
  testImplementation "org.junit.jupiter:junit-jupiter-api"
```